chunk_utils.py

```
@@ -0,0 +1,209 @@
+    id: str
+    content: str
+    metadata: Dict[str, Any]
+    source_url: str
+    created_at: str
+    chunk_type: str
+
+
+class ContentChunker:
+    """Handles chunking of content for RAG systems."""
+    
+    def __init__(self, chunk_size: int = 1000, chunk_overlap: int = 200):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)